src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/x64/conv_concat.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Why we have to make those instances x86 specific?

Reply:
Reference implementation (`ref_any`) does not support nchw/ndchw layouts on ARM. I've got the error `No supported primitive descriptors matched the provided input / output memory format filters` while pushing these layouts into the test.
nspc works well for reference on both ARM and x64. Can we replace nchw/ndchw with nhwc/ndhwc and keep conv tests common?